jsonnetsecure/jsonnet.go

```diff
@@ -111,8 +111,15 @@ func (importer *ErrorImporter) Import(importedFrom, importedPath string) (conten
 func JsonnetTestBinary(t testing.TB) string {
 	t.Helper()
 
+	// We can force the usage of a given jsonnet executable.
+	// Useful to test different versions, or run the tests under wine.
+	if s := os.Getenv("ORY_JSONNET_PATH"); s != "" {
+		return s
+	}
+
 	var stderr bytes.Buffer
-	outPath := path.Join(t.TempDir(), "jsonnet")
+	// Using `t.TempDir()` results in permissions errors on Windows, sometimes.
+	outPath := path.Join(os.TempDir(), "jsonnet")
```

Comment:
`t.TempDir()` is deleted after the test runs, but `os.TempDir()` is not. Do we really need Windows compatibility?